### ISSUE TITLE
'OcrLocator' object has no attribute 'configuration'

### ISSUE THREAD

**Issue by neanias** [robocorp/rpaframework#1206]
I'm seeing this error when trying to use the `ocr` locator with rpaframework 29.0.0. (e.g. `desktop.click('ocr:"Click me"')`). I think it's caused by this commit 283bb928b26a4c768cc037b572d172eaf73fb7ba where the `OcrLocator` class had a `configuration` attribute added (and used elsewhere), but a new version of `rpaframework-core` wasn't published at the same time with the new attribute.

This is running on Windows Server 2022.

<details>
<summary>Full traceback</summary>

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[6], line 1
----> 1 sleep(2); desktop.click('ocr:"Medicine Management"')

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\Desktop\keywords\mouse.py:134, in MouseKeywords.click(self, locator, action)
    131 action = to_action(action)
    133 if locator:
--> 134     match = self.ctx.wait_for_element(locator)
    135     self._click(action, match)
    136 else:

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\Desktop\keywords\finder.py:352, in FinderKeywords.wait_for_element(self, locator, timeout, interval)
    350 start = time.time()
    351 try:
--> 352     return self.find_element(locator)
    353 except (ElementNotFound, MultipleElementsFound) as err:
    354     error = err

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\Desktop\keywords\finder.py:308, in FinderKeywords.find_element(self, locator)
    293 @keyword
    294 def find_element(self, locator: LocatorType) -> Geometry:
    295     """Find an element defined by locator, and return its position.
    296     Raises ``ElementNotFound`` if` no matches were found, or
    297     ``MultipleElementsFound`` if there were multiple matches.
   (...)
    306         Log    Found logo at ${match.right}, ${match.top}
    307     """
--> 308     matches = self.find_elements(locator)
    310     if not matches:
    311         raise ElementNotFound(f"No matches found for: {locator}")

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\Desktop\keywords\finder.py:291, in FinderKeywords.find_elements(self, locator)
    289     return self._find(Undefined(), locator)
    290 else:
--> 291     return self._resolver.dispatch(str(locator))

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\core\locators\syntax.py:260, in Resolver.dispatch(self, locator)
    257     return literal.parse(value, self.database)
    259 root = SyntaxParser().parse(locator=locator, parser=from_literal)
--> 260 return sorted(self._resolve(root))

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\core\locators\syntax.py:273, in Resolver._resolve(self, value)
    271 for type_, func in resolvers.items():
    272     if isinstance(value, type_):
--> 273         return func(value)
    275 raise RuntimeError(f"Unexpected value: {value}")

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\core\locators\syntax.py:325, in Resolver._locator(self, locator)
    323 def _locator(self, locator: Locator):
    324     """Resolve final values for locator."""
--> 325     return self.finder(self.current, locator)

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\Desktop\keywords\finder.py:105, in FinderKeywords._find(self, base, locator)
    103 for klass, finder in finders.items():
    104     if isinstance(locator, klass):
--> 105         return finder(base, locator)
    107 raise NotImplementedError(f"Unsupported locator: {locator}")

File ~\AppData\Local\pypoetry\Cache\virtualenvs\robotframework-test-F3dnjsd3-py3.10\lib\site-packages\RPA\Desktop\keywords\finder.py:205, in FinderKeywords._find_ocr(self, base, locator)    203 confidence = locator.confidence or self.confidence
    204 language = locator.language
--> 205 configuration = locator.configuration
    206 self.logger.info(
    207     "Searching for text '%s' (region: %s, confidence: %.1f, "
    208     "language: %s, configuration: %s)",
   (...)
    213     configuration or "Not set",
    214 )
    216 def finder(image: Image.Image) -> List[Region]:

AttributeError: 'OcrLocator' object has no attribute 'configuration'
```

</details>

**Comment by mikahanninen** [comment]
@neanias Thank you for the report! Yeah I missed this one. I will make a fix release as soon as possible.

**Comment by neanias** [comment]
Thank you!

**Comment by mikahanninen** [comment]
This is now fixed with `rpaframework-recognition==5.3.7`
